spec/lib/rack/zipkin-tracer_spec.rb

```diff
@@ -75,30 +79,48 @@ def expect_host(host)
   context 'Using Rails' do
     subject { middleware(app) }
 
-    context 'accessing a valid URL of our service' do
+    context 'accessing a valid URL "/" of our service' do
       before do
-        allow(middleware(app)).to receive(:routable_request?).and_return(true)
```

Comment:
Previously, if I changed this to and_return(false), I would still see routable_request?(env) return true in rack/zipkin-tracer.rb